Detections/SymantecProxySG/ExcessiveDeniedProxyTraffic.yaml

```diff
@@ -0,0 +1,26 @@
+id: 7a58b253-0ef2-4248-b4e5-c350f15a8346
+name: Excessive Denied Proxy Traffic
+description: |
+  'This alert creates an incident when a client generates an excessive amounts of denied proxy traffic.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Syslog
```

Comment:
connectorId should be the data concector ID which is SymantecProxySG

Reply:
Corrections made and pushed

Reply:
This still shows requiredDataConnectors:
  - connectorId: Syslog
- datatypes: blank
Please fix this.

Reply:
corrections made and pushed